src/ucp/core/ucp_ep.c

```diff
@@ -873,7 +917,10 @@ ucs_status_ptr_t ucp_ep_close_nb(ucp_ep_h ep, unsigned mode)
                                     ucp_ep_close_flushed_callback, "close");
 
     if (!UCS_PTR_IS_PTR(request)) {
-        if (ucp_ep_is_cm_local_connected(ep)) {
+        if ((ucp_ep_get_cm_lane(ep) != UCP_NULL_LANE) &&
+            (mode == UCP_EP_CLOSE_MODE_FORCE)) {
+            ucp_ep_destroy_cm_lane(ep);
```

Comment:
why need special case?
seems like ucp_ep_disconnected(), when it calls ucp_ep_cleanup_lanes(), should also destroy cm lane?